gwdetchar/io/tests/test_html.py

```diff
@@ -29,6 +29,7 @@
 from MarkupPy import markup
 from pytz import reference
 from unittest import mock
+from pygments import __version__ as pygments_version
```

Comment:
Since someone went to the trouble of importing things alphabetically with some grouping, I recommend preserving that for readability (ie move this line up two lines).

Reply:
Done